crimsobot/cogs/mystery.py

```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
+    @commands.cooldown(3, 300, commands.BucketType.user)
+    async def one(self, ctx: commands.Context, spread: str = 'one') -> None:
+        """ One card """  # this also needs to be improved
+
+        fp, descriptions = await tarot.reading(spread)
+        filename = 'reading.png'
+        f = discord.File(fp, filename)
+
+        embed = c.crimbed(
+            title="{}'s reading".format(ctx.author),
+            descr=None,
+            attachment=filename,
+            footer='Type ">tarot card" for more on a specific card.',
+        )
+
+        for card_tuple in descriptions:
```

Comment:
This seems wasteful again. I think I understand why this was done, but a loop when only one element is required seems silly.

Reply:
Fixed with 328d54e